Appraisals

```diff
@@ -0,0 +1,11 @@
+appraise 'rails_4' do
+  gem 'rails', '~> 4.0'
+end
+
+appraise 'rails_5.0' do
+  gem 'rails', '~> 5.0.0'
+end
+
+appraise 'rails_5.1' do
```

Comment:
What about Rails 5.2 + Rails 6.0?

Reply:
They weren't here before, added in #104.